src/shared/utils/config-utils/get-vendor-service-config-rows.test.ts

```diff
@@ -3,14 +3,14 @@ import { getVendorServiceConfigRows } from "./get-vendor-service-config-rows";
 jest.mock("../s3", () => ({
   ...jest.requireActual("../s3"),
   fetchS3: async () => {
-    return `vendor_name,vendor_regex,vendor_id,service_name,service_regex,event_name
-    Vendor One,Vendor.*[One|1],vendor_testvendor1,Passport check,Passport.*check,EVENT_1
-    Vendor One,Vendor.*[One|1],vendor_testvendor1,Fraud check,Fraud.*check,EVENT_3
-    Vendor Two,Vendor.*[Two|2],vendor_testvendor2,Passport check,Passport.*check,EVENT_1
-    Vendor Two,Vendor.*[Two|2],vendor_testvendor2,Kbv check,Kbv.*check,EVENT_7
-    Vendor Three,Vendor.*[Three|3],vendor_testvendor3,Passport check,Passport.*check,EVENT_1
-    Vendor Three,Vendor.*[Three|3],vendor_testvendor3,Address check,Address.*check,EVENT_6
-    Vendor Four,Vendor.*[Four|4],vendor_testvendor4,Passport check,Passport.*check,EVENT_1`;
+    return `vendor_name,vendor_id,service_name,service_regex,event_name
+    Vendor One,vendor_testvendor1,Passport check,Passport.*check,EVENT_1
+    Vendor One,vendor_testvendor1,Fraud check,Fraud.*check,EVENT_3
+    Vendor Two,vendor_testvendor2,Passport check,Passport.*check,EVENT_1
+    Vendor Two,vendor_testvendor2,Kbv check,Kbv.*check,EVENT_7
+    Vendor Three,vendor_testvendor3,Passport check,Passport.*check,EVENT_1
+    Vendor Three,vendor_testvendor3,Address check,Address.*check,EVENT_6
+    Vendor Four,vendor_testvendor4,Passport check,Passport.*check,EVENT_1`;
```

Comment:
does this mean we can ditch `vendor_regex` in the config too? I guess it was only for matching against the invoices